substratevm/src/com.oracle.svm.core.posix/src/com/oracle/svm/core/posix/JavaNetNetworkInterface.java

```diff
@@ -184,6 +186,39 @@ public void toLog(Log log) {
         }
     }
 
+    static boolean isUp0(String name, int ind) throws SocketException {
```

Comment:
This looks like a translation of http://hg.openjdk.java.net/jdk8u/jdk8u/jdk/file/15a6dced59c3/src/solaris/native/java/net/NetworkInterface.c#l479.

It would be good (for people who are trying to review it now, or diagnose bugs later) to have a reference to the JDK code, probably even to the level of inter-lining the C code.